frontend/src/modules/Shared/Stack/Stack.js

```diff
@@ -96,11 +100,36 @@ export const Stack = (props) => {
       .catch((e) => dispatch({ type: SET_ERROR, error: e.message }));
   };
 
```

Comment:
Instead of re-creating the logic of the backend in the frontend, we could do the same as in the `frontend/src/modules/Shares/views/ShareView.js` (for share.canViewLogs) and use a resolver in the stack object

Reply:
Will do